conda/notices/types.py

```diff
@@ -24,6 +24,28 @@ class ChannelNotice(NamedTuple):
     interval: Optional[int]
 
 
+class ChannelNoticeSet(NamedTuple):
```

Comment:
The name is a little misleading since it's not a Python set after all, could we use this instead?

```suggestion
class ChannelNotices(NamedTuple):
```

Reply:
My only concern is that the name is too similar to the above data type `ChannelNotice`.

In defining this, I was trying to mirror what Django does with its `QuerySet` object. For notices, this is the object getting returned from our `retrieve_notices` function, so it is in a sense a set of query results. How about, `ChannelNoticeResultSet` to be more clear and avoid confusion?